Alignment/OfflineValidation/plugins/TrackerGeometryCompare.cc

```diff
@@ -35,6 +35,7 @@
 #include "CondFormats/AlignmentRecord/interface/GlobalPositionRcd.h"
 #include "CondFormats/Alignment/interface/DetectorGlobalPosition.h"
 #include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
+#include "DetectorDescription/DDCMS/interface/DDCompactView.h"
```

Comment:
Unfortunately that's at least 2 layers deep in some of these other headers (I never found it, but it has to be there otherwise it wouldn't compile/run) - this plugin clearly could be cleaned up, but I think we will do that outside of this PR